docs/_plugins/insert_token.rb

```diff
@@ -1,6 +1,6 @@
 module InsertToken
     def insert_token(input)
-        input.sub("<script>", "<script>\nmapboxgl.accessToken = '<your access token here>';")
+        input.sub("<script>", "<script>\n  mapboxgl.accessToken = '<your access token here>';")
```

Comment:
Why did you add 2 spaces here?

The formatting of our examples suggests this line should not be indented. For example, see https://www.mapbox.com/mapbox-gl-js/examples